lib/livebook_web/live/hub/edit/personal_component.ex

```diff
@@ -11,17 +11,28 @@ defmodule LivebookWeb.Hub.Edit.PersonalComponent do
     socket = assign(socket, assigns)
     changeset = Personal.change_hub(assigns.hub)
     secrets = Hubs.get_secrets(assigns.hub)
+    [_local | file_systems] = Hubs.get_file_systems(assigns.hub)
```

Comment:
It would be safer to `Enum.reject` just so we don't make an assumption on the first element :)

Reply:
maybe then pass an option to Hubs.get_file_systems?

```suggestion
    file_systems = Hubs.get_file_systems(assigns.hub, local: false)
```

?

Reply:
`local: false` or `hub_only: true` sounds good!